### ISSUE TITLE
healPower  stat

### ISSUE THREAD

**Issue by DemICE** [L2J/L2J_DataPack#198]
In this project, only healEffect exists, which is for the power of recieved heals. 

There are skills, that rather than increasing the power of recieved heals, are meant to increase the power of the heal the actor does to others (Divine power, Enlightenment -Healer, Dynasty robe set - healer).

Obviously all other afformentioned skills, also used the healEffect stat, which made them not work the way they were meant to.

I created a working healPower stat, and if you like it being implemented in this fashion, you can add it to your repositories like this:

Server side --> gameserver\model\stats\Stats.java
add ```	HEAL_POWER("healPower"),	``` under HEAL_EFFECT prefferably for clarity

Datapack side --> effecthandlers\Heal.java
add ``` 			amount = activeChar.calcStat(Stats.HEAL_POWER, amount, null, null);	``` under  ```			amount = target.calcStat(Stats.HEAL_EFFECT, amount, null, null);```

Considering that among other things too, this stat existed at an old hi5 revision i had used at 2014,  this must be a new project started from scratch you are working on right now, eh ?

**Comment by Zealar** [comment]
Yes there is bug but in retail same stats is used. When stat is mul working only on you when is add increase your power. Guess we can add it like separate effect for made it more abstract.

**Comment by DemICE** [comment]
so when a healer in oficial used enlightenmeant, he increased the power of his heals only for himself? O.o   thats something i didnt know ;o

**Comment by Zealar** [comment]
Yes only for himself
